x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/rule_creation/common_flows.cy.ts

```diff
@@ -67,6 +68,7 @@ describe('Common rule creation flows', { tags: ['@ess', '@serverless'] }, () =>
   it('Creates and enables a rule', function () {
     cy.log('Filling define section');
     importSavedQuery(this.timelineId);
+    fillRequiredFields();
```

Comment:
Are we actually check in tests what exaclty we are add to the rule?

Are we want check on Cypress level also warnings?

Reply:
> Are we actually check in tests what exaclty we are add to the rule?

Currently we don't. But I agree we should at least check what we send in request. We have discussed this in one of our Tech Sync meetings and decided that this should be done as a part of a greater test refactoring effort.

> Are we want check on Cypress level also warnings?

Since we now try to reduce the number of new Cypress tests we write, I only tested the warnings on the component level by passing different props to it and checking what has rendered. 

I was considered writing a couple of integration-level Jest tests where I would mock part of the form (index patterns and required fields) and then try to add / remove index patterns by typing / clicking. Ultimately I decided that it's not worth the effort since it's tested thoroughly on the component level. wdyt?

Reply:
@nikitaindik 
> But I agree we should at least check what we send in request. 

I don't think we need to test what send in request. We just need to assert that these fields are saved and displayed on rule details. That would ensure typed fields are saved correctly within a rule and displayed as expected.

Reply:
@nikitaindik , what your thoughts on testing saved fields, displaying in details, instead of asserting request?